templates/partners/channel-and-reseller.html

```diff
@@ -0,0 +1,132 @@
+
+{% block content %}
+<section class="p-strip--image is-dark">
+  <div class="p-strip--suru-top">
+    <div class="row">
+      <div class="col-8">
+        <h1>Channel&sol;Reseller Programme</h1>
+        <p>Canonical works with channel and reseller partners around the world to ensure that Canonical offerings are available for their customers. Additionally these partners can both resell services and value-added offerings around our technology enabling them to grow their business.</p>
+        <p>Canonical is an industry leader in OpenStack private clouds, the reference platform for all major public clouds, and the open source software operating system that runs from the desktop and all internet connected things. Our success helps our partners build a business around our technology.</p>
+        <a href="/partners/become-a-partner" class="p-button--positive">Apply to become a partner</a>
+      </div>
+      <div class="col-4 u-hide--small u-align--center u-vertically-center">
+        <img src="https://assets.ubuntu.com/v1/80472358-Canonical-reseller-icon.svg" alt="">
+      </div>
+    </div>
+  </div>
+</section>
+<style>
+  .p-strip--suru-top {
+    background-image:
+      linear-gradient(to bottom right, rgba(210, 180, 202, 0.05) 0%, rgba(210, 180, 202, 0.05) 49.9%, rgba(210, 180, 202, 0) 50%, rgba(210, 180, 202, 0) 100%),
+      linear-gradient(to top right, rgba(74, 24, 60, 0) 0%, rgba(74, 24, 60, 0) 49.9%, rgba(74, 24, 60, 0.34) 50%, rgba(74, 24, 60, 0.34) 100%),
+      linear-gradient(to top left, #f7f7f7 0%, #f7f7f7 49%, transparent 50%),
+      linear-gradient(230deg, #E95420 0%, #772953 33%, #2C001E 72%);
+    background-position: left top, right top, right bottom;
+    background-repeat: no-repeat;
+    background-size: 70% 80%, 70% 100%, 110% 20%, 100% 100%;
+    padding-bottom: 6rem;
+    padding-top: 3rem;
+  }
+
+  @media (max-width: 620px) {
+    .p-strip--suru-top {
+      background-image:
+        linear-gradient(to top right, rgba(74, 24, 60, 0) 0%, rgba(74, 24, 60, 0) 49.9%, rgba(74, 24, 60, 0.4) 50%, rgba(74, 24, 60, 0.4) 100%),
+        linear-gradient(to top left, #f7f7f7 0%, #f7f7f7 49%, transparent 50%),
+        linear-gradient(to bottom right, rgba(210, 180, 202, 0.05) 0%, rgba(210, 180, 202, 0.05) 49.9%, rgba(210, 180, 202, 0) 50%, rgba(210, 180, 202, 0) 100%),
+        linear-gradient(120deg, #2C001E 4%, #4C193E 100%);
+      background-position: right top, bottom right, top left, top left, center right;
+      background-repeat: no-repeat;
+      background-size: 100% 100%, 105% 25%, 70% 80%, 100% 100%;
+    }
+  }
+</style>
+<section class="p-strip--light is-shallow" style="margin-top: -1px;">
+  <div class="row">
+    <div class="col-8">
+      <p>Interested in becoming an Ubuntu partner? Talk to us today to learn more about our programmes.</p>
+    </div>
+    <div class="col-4">
+      <a href="/partners/become-a-partner" class="p-button--positive">Get in touch</a>
```

Comment:
It's weird that there are two links to the same form in the header.

Reply:
I removed one link